src/algebra/invertible.lean

```diff
@@ -155,6 +155,17 @@ def invertible.copy [monoid α] {r : α} (hr : invertible r) (s : α) (hs : s =
   inv_of_mul_self := by rw [hs, inv_of_mul_self],
   mul_inv_of_self := by rw [hs, mul_inv_of_self] }
 
+theorem commute.inv_right_invertible [monoid α] {a b : α} [invertible b] (h : commute a b) :
```

Comment:
```suggestion
theorem commute.inv_of_right [monoid α] {a b : α} [invertible b] (h : commute a b) :
```
since `inv_of` seems to be the name for this operator